datahub-web-react/src/app/entity/shared/components/styled/StyledTag.tsx

```diff
@@ -6,7 +6,7 @@ const generateColor = new ColorHash({
     saturation: 0.9,
 });
 
-export const StyledTag = styled(Tag)<{ $colorHash?: string }>`
+export const StyledTag = styled(Tag)<{ $colorHash?: string | null }>`
```

Comment:
I don't think that colorHash is what we want here - We need the styled tag to have the exact same color as what is the field of colorHex inside of the properties object